meta-balena-common/conf/distro/include/balena-os.inc

```diff
@@ -71,6 +71,9 @@ INITRAMFS_TASK = ""
 PACKAGE_CLASSES ?= "package_ipk"
 
 # Default the docker storage driver to aufs
+#
+# NOTE: Machine overriding performed by commit a97dd869f8
+# should be removed when default storage is switched to overlay2.
```

Comment:
@robertgzr what are the implications of changing the defaults to overlay2? As far as I can tell, the only concern are rollbacks so we should make sure that kernel support still has both overlay2 and aufs after the change.

@acostach just thinking that it might just be cleaner to change the default now, I don't think anything is holding us back.

Reply:
@alexgg why would we be concerned if the new kernel has aufs support in case of rollbacks? we are aiming to have the kernel in /boot anyway so the old kernel will be in the rollbacked OS

Reply:
Thanks @alexgg , I updated this and switched the default to overlay2.

Reply:
@floion the applications will  have migrated, so the old kernel needs to have both aufs and overlay2 support. Before we merge the change rollbacks would need to be tested anyway (it's done in the automation so it should be easy).